puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/source/foreman-installer.groovy

```diff
@@ -33,10 +36,20 @@ pipeline {
                 run_test(ruby: '2.5', puppet: '6.3')
             }
         }
-        stage("Release Nightly Package") {
+        stage('Build and Archive Source') {
+            steps {
+                dir(project_name) {
+                    git url: "https://github.com/theforeman/${project_name}", branch: foreman_branch
```

Comment:
We already have a git clone so this may not be needed? I don't mind having a clean directory though.

Reply:
The current code requires it be in a directory named for the project. I could change that code before, but I wanted to instead adapt jobs to this pattern then do re-factoring.

Reply:
I noticed a similar thing and had the same thought: good improvement, not now.